### ISSUE TITLE
Onde está o slide da aula 04

### ISSUE THREAD

**Issue by lucianagferreira** [gustavoguanabara/git-github#2725]
Procurei os slides e não encontrei, alguém pode ajudar?

**Comment by lucianagferreira** [comment]
Esperando alguém responder

**Comment by lucianagferreira** [comment]
Estou apredendo linguagem **markdown** no curso em vídeo de Git e github
# Aula de linguagem markdown 
***
Podemos __** misturar **__ configurações
## Titulo
### Titulo

Lista numerada

1. teste 1
1. teste 2

Lista demarcada
* teste

Lista de tarefas
- [x] Criar página principal 

![teste da imagem](https://github.com/gustavoguanabara/git-github/assets/145069929/a6203ef6-9738-4a45-a70f-43e7fb271544)

[Acesse esta aula](https://www.youtube.com/watch?v=LntSB-gl-ZI&list=PLHz_AreHm4dm7ZULPAmadvNhH6vk9oNZA&index=10)

Tabela

Num | Nome | Nota
--- | --- | ---
1 | Luciana | 8,5

Não entendo o comando `document.getElementById( )´ da linguagem JavaScript

Olha meu programa em Python 
```
num = int(input("Digite um valor: ")
if  num % 2 == 0
  print = "O valor é par"
```

Hello world :earth_asia:

**Comment by LoiceLipocheSitole** [comment]
Eu procurando a aula 4 em 2023 kkkkkkkkkkk

**Comment by DevRamonRigonato** [comment]
> Procurei os slides e não encontrei, alguém pode ajudar?

Olá por acaso seriam esses? [Clique aqui](https://github.com/gustavoguanabara/git-github/tree/master/slides-aulas)

**Comment by LoiceLipocheSitole** [comment]
Fazia parte da aula escrever isso! obrigada @DevRamonRigonato

**Comment by edibispo** [comment]
Onde está os slide 4

**Comment by Viniciuss2004** [comment]
Aula 4 já foi disponível a 3 anos atrás! kkkkk

**Comment by lucianagferreira** [comment]
> > Procurei os slides e não encontrei, alguém pode ajudar?
> 
> Olá por acaso seriam esses? [Clique aqui](https://github.com/gustavoguanabara/git-github/tree/master/slides-aulas)

Sim, isto faz parte do curso git e github, por isso criei a issue, Obrigada

**Comment by lucianagferreira** [comment]
> Aula 4 já estava disponível há 3 anos! kkkkk

É um exercício do curso, por isso criei a issue hahah

**Comment by edibispo** [comment]
> Aula 4 já foi disponível a 3 anos atrás! kkkkk

I know my dude...😂

**Comment by edibispo** [comment]
Eu sei pessoal...eu fiz eu só para dar resenhas..kkkk

**Comment by edibispo** [comment]
Eu sei pessoal...eu fiz eu só para dar resenhas..kkkk

**Comment by LuisFNascimento** [comment]
Caro gafanhoto, não tem slide!

**Comment by GRodrigues10** [comment]
kkkkkkkkkkkk
